src/tests/unit/tests/background/stores/card-selection-store.test.ts

```diff
@@ -231,6 +231,23 @@ describe('CardSelectionStore Test', () => {
         createStoreForCardSelectionActions('collapseAllRules').testListenerToBeCalledOnce(initialState, expectedState);
     });
 
+    test('expandAllRules', () => {
+        initialState.rules['sampleRuleId1'].isExpanded = true;
+        initialState.rules['sampleRuleId1'].cards['sampleUid1'] = true;
+
+        expectedState.rules['sampleRuleId1'].isExpanded = true;
+        expectedState.rules['sampleRuleId1'].cards['sampleUid1'] = true;
+        expectedState.rules['sampleRuleId2'].isExpanded = true;
+
+        createStoreForCardSelectionActions('expandAllRules').testListenerToBeCalledOnce(initialState, expectedState);
+    });
+
+    test('toggleVisualHelper on - no card selection or rule expansion changes', () => {});
```

Comment:
I think either you should implement the function in this PR or remove the empty tests/functions.

From a reader's perspective, seeing this in master branch would be quite confusing.

Reply:
Removed